app/(main)/league/[leagueId]/entry/[entryId]/week/Week.tsx

```diff
@@ -160,15 +161,18 @@ const Week = ({ entry, league, NFLTeams, week }: IWeekProps): JSX.Element => {
       }
       
       setEntryName(currentEntry.name);
-      let entryHistory = currentEntry?.selectedTeams || [];
+      const entryHistory = currentEntry?.selectedTeams || [];
```

Comment:
Why did this function need to be refactored for this ticket?

Reply:
Previously, we only stored URLs for team logos in the `pickHistory` state. But we need to hold on to the team name as well to use it as the `alt` text of the image. That why the `entryHistoryObject`.